Lib/test/test_descr.py

```diff
@@ -4632,6 +4630,15 @@ def check(expr, x, y):
             check(expr, a, a)
             check(expr, a, N1)
             check(expr, a, N2)
+            B = type('B', (), {rname: specialmethod})
```

Comment:
Please add a comment to explain how B is different from A. "name" vs "rname" is not obvious when I read the code.